src/App.js

```diff
@@ -1,13 +1,35 @@
-import logo from "./logo.svg";
-import "./App.css";
-
+import React, { useState } from "react";
 function App() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
```

Comment:
can you using multiple form instead of repeating your self.
For example:
``` bash
const [person,setPerson] = useState({firstName:"",lastName:""})
```